tests/acceptance-tests/wayland/CMakeLists.txt

```diff
@@ -113,6 +113,10 @@ set(EXPECTED_FAILURES
 
   # Not yet implemented, see https://github.com/MirServer/mir/issues/2861
   XdgPopupTest.when_parent_surface_is_moved_a_reactive_popup_is_moved
+
+  # Should throw an exception, but we log instead because of
```

Comment:
The exception is an implementation detail. We should report an "orphaned" protocol violation